index.html

```diff
@@ -1300,65 +1302,43 @@ <h2>
         <h2>
           <dfn>canMakePayment()</dfn> method
         </h2>
+        <div class="note" title="canMakePayment() vs hasEnrolledInstrument()">
+          <p>
+            The <a>canMakePayment()</a> method returns a <a>Promise</a>
+            that is fulfilled with true if the <a>user agent</a> supports
+            any of the <a data-link-for="PaymentMethodData">supportedMethods</a>
+            supplied to the <a>PaymentRequest</a>
+            <a data-lt="PaymentRequest.PaymentRequest()">constructor</a>, or
+            false if none are supported.
+          </p>
+          <p>
+            A true result from <a>canMakePayment()</a> does not imply that the
+            user has an provisioned instrument ready for payment. For that, use
+            <a>hasEnrolledInstrument()</a> instead.
+          </p>
+        </div>
+        <p data-tests="payment-request-canmakepayment-method.https.html">
+          The <a>canMakePayment()</a> method MUST run the <a>can make payment
+          algorithm</a> with <var>checkForInstruments</var> set to false.
+        </p>
+      </section>
+      <section data-dfn-for="PaymentRequest" data-link-for="PaymentRequest">
+        <h2>
+          <dfn>hasEnrolledInstrument()</dfn> method
+        </h2>
         <p class="note">
-          The <a>canMakePayment()</a> method can be used by the developer to
-          determine if the <a>PaymentRequest</a> object can be used to make a
-          payment, before they call <a>show()</a>. It returns a <a>Promise</a>
-          that will be fulfilled with true if the <a>user agent</a> supports
-          any of the desired <a>payment methods</a> supplied to the
-          <a>PaymentRequest</a> constructor, and false if none are supported.
-          If the method is called too often, the user agent might instead
-          return <a>a promise rejected with</a> a "<a>NotAllowedError</a>"
-          <a>DOMException</a>, at its discretion.
+          The <a>hasEnrolledInstrument()</a> method can be used by the developer
+          to determine if the <a>user agent</a> not only has support for one of
+          the desired <a>payment methods</a> but is also "ready for payment" (
```

Comment:
I'd suggest dropping everything after "but also..." and replacing it with:

"and if a <a>payment handler</a> has an instrument ready for payment. See <a href="#"hasEnrolledInstrument-privacy"><a> for privacy considerations."

And also drop the actual details (i.e., the "If the method is called too often...").

Reply:
Done. I also updated the corresponding note in canMakePayment() section above to match the wording, and also refer to the privacy section.